control-plane/agents/core/src/volume/tests.rs

```diff
@@ -19,6 +19,7 @@ use common_lib::{
 };
 use testlib::{Cluster, ClusterBuilder};
 
+use common_lib::types::v0::message_bus::VolumeStatus;
```

Comment:
move this above together with the other common_lib uses?

Reply:
Done.